source/codegen/common_helpers.py

```diff
@@ -19,6 +19,13 @@ def is_enum(parameter):
 def is_struct(parameter):
   return parameter["type"].startswith("struct")
 
+
+def any_function_uses_timestamp(functions):
```

Comment:
What do you think about replacing this with some stuff in config.py for each driver. So the codegen doesn't have the types coded in?

additional_headers: ["custom/nidaqmx_conversions.h"],
types_require_conversion: ["CVIAbsoluteTime"]

I like keeping `convert_to_grpc` as a standard convention for naming converters.

Reply:
Oh, actually. I like this logic for the `import "google/protobuf/timestamp.proto";` but not as much for the `#include "custom/nidaqmx_conversions.h"`